enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -224,6 +224,19 @@ class SubsidyAccessPolicyCanRedeemRequestSerializer(serializers.Serializer):
     )
 
 
+# pylint: disable=abstract-method
+class SubsidyAccessPolicyDeleteRequestSerializer(serializers.Serializer):
+    """
+    Request Serializer for DELETE parameters to an API call to delete a subsidy access policy.
+
+    For view: SubsidyAccessPolicyViewSet.destroy
+    """
+    reason = serializers.CharField(
```

Comment:
nit: might be worth adding `allow_null=True` here. https://www.django-rest-framework.org/api-guide/fields/#allow_null

Reply:
Oh, i totally forgot to write unit tests for the reason field.